policyengine_us/parameters/gov/states/ms/tax/income/exemptions/dependents/amount.yaml

```
@@ -0,0 +1,12 @@
+description: Mississippi provides an exemption of this amount per dependent.
+metadata:
+  unit: currency-USD
+  label: Mississippi dependent exemption
+  reference: 
+    - title: Mississippi State FORMS 80-105 AND 80-205
+      href: https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=5
+    - title: Miss. Code Ann. § 27-7-21 (e)
+      href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=adb5f365-0449-4d49-80f0-22673c0d0165&nodeid=AAPAAFAABAAP&nodepath=%2fROOT%2fAAP%2fAAPAAF%2fAAPAAFAAB%2fAAPAAFAABAAP&level=4&haschildren=&populated=false&title=%C2%A7+27-7-21.+Exemptions+allowed.&config=00JABhZDIzMTViZS04NjcxLTQ1MDItOTllOS03MDg0ZTQxYzU4ZTQKAFBvZENhdGFsb2f8inKxYiqNVSihJeNKRlUp&pddocfullpath=%2fshared%2fdocument%2fstatutes-legislation%2furn%3acontentItem%3a8V3T-78H2-D6RV-H4WK-00008-00&ecomp=8gf5kkk&prid=e1d2bafd-2018-44e8-a4fc-e9e2b08a4daa
+values:
+  2022-01-01: 1_500
```

Since the legal code says the legislature most recently revised it in 2019
```suggestion
  2020-01-01: 1_500
```

---

Got it! Thanks for pointing that out